src/smart-app-banner.js

```diff
@@ -0,0 +1,181 @@
+var extend = require('xtend/mutable');
```

Comment:
Not convinced that replacing a file with a folder with the file enhances readability. We don’t have and not planning to spawn tenths of various modules, so until there is no tangible reason for that let’s keep things simple